module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -378,4 +378,27 @@ public function findInThemes($relativePathPattern)
         }
         return array_values($results);
     }
+
+    /**
+     * Search vendor directory for a file.
+     *
+     * @param string $filePath Relative path to file
+     * @param string $fileType Specified file extension
+     *
+     * @return array|null
+     */
+    public function findInPackage($filePath, $fileType = 'js')
+    {
+        if (strpos($filePath, 'vendor') !== false
```

Comment:
Should we put slashes around vendor (and/or add additional identifying path parts) to make this less potentially ambiguous?